### DISCUSSION TITLE
The Url field doesn't render.

### DISCUSSION THREAD

**Discussion by havana7** [OrchardCMS/OrchardCore#13338] [Q&A] [answered]
HI!

Why can't I see the Url field? Where is the mistake? Thank you.

`<h1>{{ Model.ContentItem.DisplayText }}</h1>
{% for project in Model.ContentItem.Content.Projects.ContentItems %}
    <h4>{{ project.DisplayText }}</h4>
    <p>{{ project.HtmlBodyPart.Html }}</p>
    <p>{{ project.Url.Text }}</p>
{% endfor %}`

**Comment by hishamco** [comment, marked as answer]
If I have `Person` content type with `ProfileLink` as `LinkField`

```
{{ Model.ContentItem.Content.Person.ProfileLink.Url }}
```

should display the profile link for the current viewed person
